### PR TITLE
Feedback for edges in draggable graph example

### PR DESCRIPTION
**Pull request by scymtym**
Update edges in addition to the node while dragging.

### CODE REVIEW

**Comment by lokedhs** [comment]
Extra indentation happens automatically in Emacs if you have `slime-indentation` enabled.

As for the loop keywords, I don't think most of the McCLIM code uses the colon prefix before the keywords.

**Comment by scymtym** [comment]
* Removed keywords in `loop` form
* Indented `do` clause

File: Examples/draggable-graph.lisp [hunk 1]
```diff
@@ -24,49 +24,39 @@
...
 (defun generate-graph (frame pane)
   (declare (ignore frame))
   (format-graph-from-roots
    (list (find-class 'number))
    (lambda (object stream)
-     (present (class-name object)
-	      (presentation-type-of object)
-	      :stream stream))
+     (present (class-name object) (presentation-type-of object)
+              :stream stream))
```
Comment (dkochmanski) [inline Examples/draggable-graph.lisp]: I don't understand why did you change one parameter per line style to "two parameters in the first and one remaining in a separate line", I'm fine with having all in one line or putting everything in separate lines but such mixed style doesn't convince me (especially here where we don't have that many arguments,  I could understand two obligatory arguments in first line and keyword arguments in next (one by one) if there were more of them.

File: Examples/draggable-graph.lisp [hunk 2]
```diff
@@ -24,49 +24,39 @@
...
-       (declare (ignore parent))       
-       (push edge edges))
-     (slot-value node 'climi::edges-to))
-    edges))
+  (append (alexandria:hash-table-values (slot-value node 'climi::edges-from))
+          (alexandria:hash-table-values (slot-value node 'climi::edges-to))))
+
+(defun node-and-edges-region (node edges)
+  (stupid-copy-rectangle
+   (reduce #'region-union edges :initial-value node)))
```
Comment (dkochmanski) [inline Examples/draggable-graph.lisp]: please put `(reduce …` in the same line as `stupid-copy-rectangle`

File: Examples/draggable-graph.lisp [hunk 3]
```diff
@@ -85,42 +75,39 @@
...
+  (let* ((stream *standard-output*)
+         (node-record (find-graph-node record))
+         (edge-records (node-edges node-record))
+         (graph-record (output-record-parent node-record))
+         (erase-region))
+    (assert (typep graph-record 'graph-output-record))
+    (erase-output-record node-record stream)
+    (drag-output-record
+     stream node-record
+     :feedback (lambda (record stream old-x old-y x y mode)
```
Comment (dkochmanski) [inline Examples/draggable-graph.lisp]: please put `stream` in the same line as the function. In general please avoid style, where first function argument gets its own line. I don't recall this style being used anywhere in the source code and I'm not a fan of it myself. If lambda function is nested too deep for feedback use flet.

File: Core/clim-core/graph-formatting.lisp [hunk 4]
```diff
@@ -491,8 +491,10 @@ Assumes that GENERATE-GRAPH-NODES has generated only nodes up to the cutoff-dept
   ((stream)
    (arc-drawer)
    (arc-drawing-options)
-   (from-node :initarg :from-node)
-   (to-node :initarg :to-node)))
+   (from-node :initarg :from-node
+              :reader from-node)
+   (to-node :initarg :to-node
+            :reader to-node)))
```
Comment (dkochmanski) [inline Core/clim-core/graph-formatting.lisp]: please put reader arguments in the same line (this style is mixed across the codebase, but this line is short enough to make it reasonable).

**Comment by scymtym** [comment]
Reformatted.

**Comment by dkochmanski** [comment]
looks good to me, thank you.

**Pull request merged**